gulp-tasks/publish-cdn.js

```diff
@@ -26,8 +26,13 @@ const handleCDNUpload = async (tagName, gitBranch) => {
   const buildFilesDir =
     await publishHelpers.groupBuildFiles(tagName, gitBranch);
 
-  logHelper.log(`Uploading '${tagName}' to CDN.`);
-  const publishUrls = await cdnUploadHelper.upload(tagName, buildFilesDir);
+  let friendlyTagName = tagName;
+  if (friendlyTagName.indexOf('v') === 0) {
```

Comment:
Changes here LGTM on the whole. I would add in a comment explaining why we're doing the index check against `v`as the hangouts history for that convo is likely to not be accessible to future contributors.

Reply:
👍 with change